appveyor.yml

```diff
@@ -113,7 +132,10 @@ for:
       - sh: "pytest -vv tests/integration"
       - sh: "pytest -vv -n 4 tests/regression"
       - sh: "/tmp/black --check setup.py tests samcli scripts"
-      - sh: "python scripts/check-isolated-needs-update.py"
+
+      # Set JAVEHOME to java11
+      - sh: "JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64"
```

Comment:
JAVA_HOME for java8 is not set. I am not sure how appveyor works, but can this be an issue on next run, when JAVA_HOME will be for java11 and test would run expecting them to be 8?

Reply:
Nevermind, now that I think about it, it shouldn't be an issue since it'll be a new session.

Reply:
We set `JAVA_HOME` to java8 on line 82 (first step in the install step). The first pass of build tests will run with java8. I needed a way to also run the Java11 build tests. Instead of creating another appveyor project, I edit the `JAVA_HOME` env var to be java11 within the test_script phase of appveyor.